lib/marked.cjs

```diff
@@ -561,35 +561,41 @@ var Tokenizer = /*#__PURE__*/function () {
 
         if (!(cap = itemRegex.exec(src))) {
           break;
-        }
+        } //lines = cap[2].split('\n');
```

Comment:
Should this comment be removed?

Reply:
This was leftover in the compiled file looks like. Recompiled so this should now be up to date.